packages/block-library/src/freeform/edit.js

```diff
@@ -83,12 +83,15 @@ function ClassicEdit( {
 		}
 
 		const editor = window.tinymce.get( `editor-${ clientId }` );
-		const currentContent = editor?.getContent();
+		if ( ! editor ) {
+			return;
+		}
 
+		const currentContent = editor.getContent();
 		if ( currentContent !== content ) {
 			editor.setContent( content || '' );
```

Comment:
Any possible concerns that this won't run once `window.tinymce.get( `editor-${ clientId }` )` is populated?